### PR TITLE
fixes #29938 - change default logging layout

### PR DESCRIPTION
**Pull request by domitea**
Complementary PR for https://github.com/theforeman/foreman/pull/7220 to make default new kind of logger mentioned in liked PR

### CODE REVIEW

**Comment by ekohl** [comment]
If we want to make this active for all existing installations, it also needs an installer migration.

**Comment by tbrisker** [comment]
Core has been merged, @domitea please add the migration as suggested

**Comment by domitea** [comment]
Migration added in https://github.com/theforeman/foreman-installer/pull/557, also Readme was affected by this change.

File: README.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ previous stable release.
 
 ### Foreman version compatibility notes
 
-This module targets Foreman 2.0+.
+This module targets Foreman 2.2+.
```
Comment (ekohl) [inline README.md:77]: We don't do this, see the support policy. Instead, we would write a new paragraph that on Foreman versions older than 2.2 you need to change the logging layout parameter to `pattern`.

**Comment by ekohl** [comment]
And you need to add the new value to the valid values in `Enum`.

**Comment by domitea** [comment]
@ekohl thanks fro mentioning enum! Readme updated.

**Comment by tbrisker** [comment]
@ekohl what's the status here?

**Review by ekohl — APPROVED** [review]

**Comment by ekohl** [comment]
It sort of slipped form my radar.

**Pull request merged**